### ISSUE TITLE
Allow to send request with Command+R

### ISSUE THREAD

**Issue by weyert** [Get-API-App/Issue-Tracker#47]
Allow to selected request to triggered to be send via Command+R

**Comment by NghiaTranUIT** [comment]
Currently, you can use Command+Enter to send the current open tab (Request).

It thought Command+Enter is easier than Command+R, isn't it?

**Comment by weyert** [comment]
Yeah, I think Paw used the other approach and thought maybe both would be possible?

**Comment by NghiaTranUIT** [comment]
Not sure if macOS allows 2 shortcuts for 1 action, but I prefer Command+Enter since it's wide-used on Postman/Insomnia/HTTPie, ...

**Comment by NghiaTranUIT** [comment]
maybe in the future, we can customize shortcuts.
